tensorboard/plugins/debugger_v2/debug_data_multiplexer.py

```diff
@@ -49,6 +55,28 @@ def run_in_background(target):
     thread.start()
 
 
+def _alert_to_json(alert):
+    # TODO(cais): Replace this with Alert.to_json() when supported by the
+    # backend.
+    global debug_events_monitors
```

Comment:
Can we directly import `debug_events_monitors` here instead of relying
on it having been implicitly set by a previous call to `Runs()`?

Reply:
Done. This will be replaced by a `to_json()` call on the Alert object anyway. So it's just temporary.

BTW, I sync'ed with @davidsoergel offline. He's okay with me merging the PR as is with only @wchargin's approval.

Reply:
Thanks for the review, @wchargin!

Reply:
SGTM; thanks.